src/platform/utilities/feature-toggles/featureFlagNames.json

```diff
@@ -231,7 +231,6 @@
   "virtualAgentUpgradeWebchat14158": "virtual_agent_upgrade_webchat_14_15_8",
   "virtualAgentEnableMsftPvaTesting": "virtual_agent_enable_msft_pva_testing",
   "virtualAgentEnableNluPvaTesting": "virtual_agent_enable_nlu_pva_testing",
-  "yellowRibbonDegreeFilter": "yellow_ribbon_degree_filter",
   "yellowRibbonEnhancements": "yellow_ribbon_mvp_enhancement",
```

Comment:
We can remove this `yellow_ribbon_mvp_enhancement` feature toggle as well, while we are hear cleaning up.

Reply:
Done

Reply:
Thank you!